playground/app.js

```diff
@@ -352,6 +352,14 @@ class App extends Component {
         <div className="col-sm-5">
           {!this.state.form ? null :
             <Form
+              id="test-id"
+              className="testclass big"
+              name="test-name"
+              method="put"
+              target="_blank"
+              autocomplete="off"
+              enctype="multipart/form-data"
+              acceptcharset="ISO-8859-1"
```

Comment:
One last bit: could you please remove the use of custom attributes from the playground? I don't really want to limit the accepted charset for instance :) I think we shouldn't add any.